ext/patches/MSToolkit/include/mzMLWriter.h

```diff
@@ -0,0 +1,135 @@
+/*

```

Comment:
Why do we need this patch again? Sorry, I'm sure I knew at one point, but I can't seem to remember now.

Reply:
We don't actually apply that file as a patch. We should just delete it. I'm not sure the other patches for MSToolkit are needed either, except for adding a CMakeLists.txt. I'm running a test now.

Reply:
The MSToolkit patches are needed because several calls to standard data types and functions are not tagged with the name space e.g. std::string. I'll submit a pull request to Mike.